wp-content/themes/workday/functions.php

```diff
@@ -32,3 +33,40 @@ function workday_stylesheets() {
 	);
 }
 add_action( 'wp_enqueue_scripts', 'workday_stylesheets', 20 );
+
+/**
+ * Output the Workday Minnesota theme specific header
+ */
+function largo_header() {
+	
+	?>
+    <div class="header-container">
+        <div class="header-container-inner">
+            <div class="nav-left">
+                <?php
+                /* Check to display Donate Button */
+                if ( of_get_option( 'show_donate_button') ) {
+                    ?>
+                        <a href="<?php echo of_get_option( 'donate_link' ); ?>">
+                            <img src="<?php echo get_stylesheet_directory_uri(); ?>/img/donate.png" class="donate-icon">
+                            <label><?php _e('Donate', 'workday'); ?></label>
+                        </a>
+                    <?php
+                }
+                ?>
+            </div>
+            <div class="nav-center">
+                <a href="<?php echo esc_url( home_url( '/' ) ); ?>"><?php largo_home_icon( 'icon-white', 'orig' ); ?></a>
```

Comment:
If this use of `largo_home_icon` is replaced with this code from Largo's `largo_header()`, we can use Largo's small/medium/large image selection options:

https://github.com/INN/largo/blob/c41d8519370b48a6e53ce36b3b516d3e54e39d66/inc/header-footer.php#L50-L53

Reply:
Good idea. Updated in https://github.com/INN/umbrella-workdayminn/pull/19/commits/c024f296e726a9688a251f6455079cf3c00c5b23.